### PR TITLE
homework from lesson 14

### PR DESCRIPTION
**Pull request by yurii-polych**
Homework 14

### CODE REVIEW

File: 14/hw-14.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+#    замінених символів
+# ======================================
+
+import json
+from re import fullmatch, sub
+
+
+# =============== Task 1. ===============
+def validate_phonenumber(phonenumber):
+    return fullmatch(r'(\+380[0-9]{9}$)|(380[0-9]{9}$)|(0[0-9]{9}$)', phonenumber)
```
Comment (acman) [inline 14/hw-14.py]: nice, but can be simplified like `(\+?380|0)\d{9}$`, play with it on https://regex101.com/

File: 14/hw-14.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            break
+        case _:
+            print('Please enter correct command.')
+
+
+# =============== Task 2. ===============
+def change_email():
+    file_name = input('Enter file name: ')
+    replaced_email = '*@*'
+    pattern = r'[\w\-\.]+@[a-zA-Z]+\.com\.ua|[\w\-\.]+@[a-zA-Z]+\.com'
```
Comment (acman) [inline 14/hw-14.py]: nice, but I think the domain can be not only `com.ua` or `com` so it could be changed like `[\.a-z]+`

**Review by acman — APPROVED** [review]
LGTM!

File: 14/hw-14.py (hunk 1 above)
Comment (yurii-polych) [reply 14/hw-14.py]: yes, your version is better.

File: 14/hw-14.py (hunk 2 above)
Comment (yurii-polych) [reply 14/hw-14.py]: yeah, but I decided to check only this two versions in this case. Should I make changes and ask for re-request?

**Pull request merged**